src/components/calcite-button/calcite-button.tsx

```diff
@@ -150,12 +134,19 @@ export class CalciteButton {
 
     return (
       <Tag
-        {...attributes}
+        aria-label={this.el.getAttribute("aria-label")}
```

Comment:
These are still just being passed from the host to the internal tag as the same attribute. Can we add props for things like 'label', 'href', 'name', 'rel' etc? Then they can be documented with descriptions on how and when they can be set.